doc/source/development.rst

```diff
@@ -44,71 +44,57 @@ Install Required Operating System Packages
-         texlive-latexmk texlive-cmap texlive-babel-english \
-         texlive-times texlive-titlesec texlive-tabulary texlive-framed \
-         texlive-float texlive-upquote texlive-parskip texlive-needspace \
-         texlive-makeindex-bin texlive-collection-fontsrecommended \
-         texlive-psnfss
+LaTeX documentation build environment
+  A full LaTeX installation is required to build the PDF documentation
+  [#f1]_.
 
-On Fedora the following commands should install the required packages:
+The above mentioned system packages will be installed by calling the
+`install_devel_packages.sh` helper script from the checked out git
+sources as follows:
 
 .. code:: shell-session
 
-   $ dnf install python3 python3-devel 'python3dist(pip)' \
-      'python3dist(tox)' make gcc which xz xorriso libxml2-devel \
-      libxslt-devel enchant genisoimage ShellCheck
-   $ # LaTeX packages
-   $ dnf install latexmk texlive-cmap texlive-metafont texlive-ec \
-      texlive-babel-english texlive-fncychap texlive-fancyhdr \
-      texlive-titlesec texlive-tabulary texlive-framed texlive-wrapfig \
-      texlive-parskip texlive-upquote texlive-capt-of texlive-needspace \
-      texlive-makeindex texlive-times texlive-helvetic texlive-courier \
-      texlive-gsftopk texlive-updmap-map texlive-dvips
+   $ sudo helper/install_devel_packages.sh
+
+.. note::
 
+   The helper script checks for the package managers `zypper` and
+   `dnf` and associates a distribution with it. If you use a
+   distribution that does not use one of those package managers
+   the script will not install any packages and exit with an
+   error message. In this case we recommend to take a look at
+   the package list encoded in the script and adapt to your
+   distribution and package manager as needed.
 
 Create a Python Virtual Development Environment
 -----------------------------------------------
 
-We recommend to setup a Python virtualenv for development. For Python 3
-:command:`python3 -m venv` is used, as it is bundled with Python itself
-(see https://docs.python.org/3/library/venv.html for details). For Python
-2.7, use :command:`virtualenv`, which is provided via :command:`pip` or as
-an extra package by your favorite Linux distribution.
-
-The following commands initializes a development environment for Python 3:
+The following commands initializes and activated a development
+environment for Python 3:
 
 .. code:: shell-session
 
-   $ python3 -m venv .env3
-   $ source .env3/bin/activate
-   $ pip install -r .virtualenv.dev-requirements.txt
-   $ python3 setup.py develop
+   $ tox -e devel
+   $ source .tox/3/bin/activate
 
-The :command:`develop` target of the :command:`setup.py` script
-automatically creates the application entry point called
-:command:`kiwi-ng-3`, which allows you to call the your modified version of
-KIWI from inside the virtual environment:
+The commands above automatically creates the application entry point
```

Comment:
I'm not sure about the term "application entry point". Technically, it's correct, but it's Python lingo. I fear, this could confuse people if someone isn't a vivid Python developer or heard about this term.

As such, I'd suggest to simplify it:

```suggestion
The commands above automatically creates the application script
```